### PR TITLE
HowTo: Set up XDebug profiling

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
Article about how to setup XDebug
Jira ticket: https://spryker.atlassian.net/browse/DOC-136
## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+description: Learn how to setup XDebug profiling in a local development environment.
+template: howto-guide-templatel
+last_updated: Jun 19, 2023
+---
+
+This document explains how to set up *XDebug profiling* in a local development environment to understand in detail the cost of functions and transactions in terms of time and memory demand. XDebug profiling lets you analyze your applications, find out the reason for performance and stability issues, and start profiling your Spryker project.
+
+## Prerequisites
+
+* Working SCOS applicaiton setup with our Docker SDK *.
```
Comment (VadymSachenko) [inline docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md:12]: What is the meaning of asterisk (*) here?
What does "our" refer to? Spryker or a person that uses the guide?
```suggestion
* Working SCOS application setup with our Docker SDK *.
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]
See comments

File: docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md (hunk 1 above)
Comment (ThomasLehnerSpryker) [reply docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md:12]: - Meant to create bullet point list
- refers to Spryker's Docker SDK as provided via github

Comment (VadymSachenko) [reply docs/scos/dev/tutorials-and-howtos/howtos/howto-setup-xdebug-profiling.md:12]: @ThomasLehnerSpryker I mean the asterisk in the end of the sentence, after "SDK"

**Review by VadymSachenko — DISMISSED** [review]

File: _data/sidebars/scos_dev_sidebar.yml [hunk 2]
```diff
@@ -3857,7 +3857,7 @@ entries:
       url: /docs/scos/dev/sdk/data-export.html
   - title: Tutorials and HowTos
     url: /docs/scos/dev/tutorials-and-howtos/tutorials-and-howtos.html
-    nested:
+    nested:
```
Comment (VadymSachenko) [inline _data/sidebars/scos_dev_sidebar.yml:3860]:
```suggestion
    nested:
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**